pyproject.toml

```diff
@@ -17,16 +17,20 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 
+[tool.poetry.scripts]
+textual = "textual.cli.cli:run"
+
 [tool.poetry.dependencies]
 python = "^3.7"
 rich = "^12.0.0"
 
 #rich = {git = "git@github.com:willmcgugan/rich", rev = "link-id"}
 typing-extensions = { version = "^3.10.0", python = "<3.8" }
+click = "8.1.2"
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.3"
-black = "^22.1.0"
+black = "^22.3.0"
```

Comment:
Had to upgrade black to [avoid it conflicting with click 8.1.2 ](https://github.com/psf/black/issues/2964)